.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "Vendor/OCMock"]
 	path = Vendor/OCMock
 	url = https://github.com/erikdoe/ocmock.git
+[submodule "Vendor/OCHamcrest"]
+	path = Vendor/OCHamcrest
+	url = https://github.com/hamcrest/OCHamcrest.git
```

Comment:
Do we need to update cgmanifest? or is the version the same?

Reply:
Updated.